mongo/client_examples_test.go

```diff
@@ -314,3 +314,50 @@ func ExampleConnect_aWS() {
 	}
 	_ = ecClient
 }
+
+func ExampleConnect_versionedAPI() {
+	// Configure a client with versioned API.
+	//
+	// Versioned API is a new feature in MongoDB 5.0 that allows user-selectable API versions, subsets of MongoDB
```

Comment:
For consistency with other examples docs, I'd suggest keeping case of structs/methods consistent with the driver API.

"client" => "Client"
"generic command" => "generic RunCommand"